backend/core/views/fill_data_modal.cfm

```diff
@@ -85,6 +85,7 @@
                                 </select>
                             </div>
                         </cfif>
+                        <input type="boolean" name="after_registration" hidden>
```

Comment:
The type "boolean" doesnt exist, please use "hidden" instead:
<input type="hidden" name="after_registration">